docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -30,3 +30,10 @@ If you see gaps for numerous rules:
 and then reactivating them in small batches at staggered intervals. This
 ensures {kib} does not attempt to run all the rules at the same time.
 * Consider adding another {kib} instance to your environment.
+
+Even if your rule runs at its scheduled time, there might still be missing alerts if your ingestion pipeline delay is greater than your rule interval + additional look-back time. As such, it is recommended that you use caution when creating custom rules with interval + additional look-back time values smaller than 6 minutes.
+
+You can also eliminate ingestion pipeline delay by specifying the `Timestamp override` field value to `event.digested` in <<rule-ui-advanced-params, advanced settings>> during rule creation. The detection engine uses the value from the `event.digested` field as the timestamp of when to execute the rule. For example, if an event occurred at 10:01 and was ingested in {es} at 10:10, the rule you created to detect that event would execute at 10:10 and generate an alert. 
```

Comment:
Clarifications:
> You can also eliminate ingestion pipeline delay by specifying the `Timestamp override` field value to `event.digested` in <<rule-ui-advanced-params, advanced settings>> during rule creation. 

You can **eliminate the risk of missed alerts due to** ingestion pipeline delay by specifying the `Timestamp override` field value to **`event.ingested`** in <<rule-ui-advanced-params, advanced settings>> during rule creation **or editing.** 

> The detection engine uses the value from the `event.digested` field as the timestamp of when to execute the rule. For example, if an event occurred at 10:01 and was ingested in {es} at 10:10, the rule you created to detect that event would execute at 10:10 and generate an alert. 

The detection engine uses the value from the **`event.ingested`** field as the timestamp **when executing** the rule. For example, if an event occurred at 10:01, **and due to a 9-minute ingestion pipeline delay, was ingested into {es} at 10:10, a rule created to detect that event, with interval + additional look-back time of say 6 minutes, even if executing at 10:12, 11 minutes after the event occurred, would still detect the event, since the `event.ingested` timestamp was only two minutes earlier, well within the 6 minute rule query.**